flagging_site/templates/api/index.html

```diff
@@ -25,4 +25,7 @@ <h2>Available Endpoints</h2>
     <ul>
         <li><a href="{{ url_for('api.reachesapi') }}" target="_blank">Flagging Model Outputs</a></li>
     </ul>
+    <ul>
+        <li><a href="{{ url_for('api.boathousesapi') }}" target="_blank">Boathouses</a></li>
+    </ul>
```

Comment:
Do this instead:

```html
    <ul>
        <li><a href="{{ url_for('api.reachesapi') }}" target="_blank">Flagging Model Outputs</a></li>
        <li><a href="{{ url_for('api.boathousesapi') }}" target="_blank">Boathouses</a></li>
    </ul>
```

`<ul>` means unordered list (i.e. bullet points), so you want to include all the `<li>`'s (list items) in the single list.

Reply:
I have replaced the pair of single-item lists with a single list containing two items.